include/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -15,6 +15,7 @@
 #endif
 
 #include <string>
+#include <vector>
```

Comment:
Compilation at runtime yes.
There is ambiguity in the attempt to differentiate containers who's elements cannot have their address taken. See the other part of the commit for what was removed/replaced.
As far as I know STL only allows this behavior for std::vector<bool>.

I'll add a FIXME, but I don't have the time to delve into the arcane semantics of **decltype** to fix it in a more efficient way (especially not knowing if that is even possible).

While I appreciate the inclusion of vector is sub-optimal in a performance sense, not including it and crashing is way worse.